### PR TITLE
Extract fly.Image in its own package

### LINKED ISSUE #47: Less globals, more require/import on v8env
The `fly.` global is sorta gross, it'd be much nicer to:

```javascript
import { Image } from 'fly/image'
```

The tricky part of this is that we build our base v8 context with webpack, and then build an app with webpack separately. So the app level webpack needs to understand how to import/require from that base webpack (or we need to do proper module loading).

### PR DESCRIPTION
**Pull request by jeromegn**
- Allows require/import of "bundled" packages
- Heavy refactor of interactions with isolated-vm from within v8 (uses globals)
- Removes the old middleware stuff from v8env

fixes #47.

### CODE REVIEW

File: v8env/bridge.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        default:
+          logger.warn("Unknown constructor, wrapping in ivm.Reference:", ctor.name)
+          return new ivm.Reference(arg);
+        // throw new Error(`Can't prepare a non-transferable value (constructor: '${ctor && ctor.name || 'unknown'}')`);
+      }
+    },
+    dispatch(name, ...args) {
+      logger.debug("dispatch", name)
+      return dispatch.apply(null, [name, ...args.map((a) => bridge.prepareValue(a))])
```
Comment (mrkurt) [inline v8env/bridge.js]: This effectively makes `dispatch` accessible. Is there any reason we wouldn't want it to be?

Comment (jeromegn) [reply v8env/bridge.js]: I was thinking about that, I think it's fine. Just got to be careful with how we handle it. I think the worse case is we would throw an error. We check the function name passed against a map and are pretty strict about it.

File: v8env/bridge.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+        case Function:
+          return bridge.wrapFunction(arg)
+
+        // simplified-values
+        case Error:
+          return arg.stack || arg.message || arg.toString()
+
+        default:
+          logger.warn("Unknown constructor, wrapping in ivm.Reference:", ctor.name)
```
Comment (mrkurt) [inline v8env/bridge.js]: Does it break stuff if this just errors? I _doubt_ wrapping random shit in ivm.Reference is a problem, but if we can just error on stuff we don't recognize that's probably safer.

Comment (jeromegn) [reply v8env/bridge.js]: I had that initially as you can see in the commented line below :)

I think it's safe to do either, might be more obvious something is going wrong if it bombs. Good call.

**Pull request merged**